src/rws_subscription.cpp

```diff
@@ -57,11 +62,14 @@ namespace abb :: rws
     // Wait for (non-ping) WebSocket frames.
     do
     {
+      auto last_frame = std::chrono::steady_clock::now();
       now = std::chrono::steady_clock::now();
```

Comment:
Calling `now()` two times in a row is strange. Also, explicitly specifying that `last_frame` is immutable makes the code easier to understand:
```suggestion
      now = std::chrono::steady_clock::now();
      auto const last_frame = now;
```